.eslintignore

```diff
@@ -2,4 +2,5 @@ node_modules
 static/js/clipboard.min.js
 venv
 env
-static/scss/libs
\ No newline at end of file
+static/scss/libs
+frontend/out
```

Comment:
I'm sorry, I snuck in an unrelated change, but this directly might be present if you've checked out the `react-experiment` branch in the past, and it's annoying if ESLint tries to lint it.